### DISCUSSION TITLE
Storage management

### DISCUSSION THREAD

**Discussion by jokerigno** [blakeblackshear/frigate#7572] [General]
Dear all,

regarding storage...I have deleted all my snapshots and in Frigate WEBUI and changed config as follows:

    record:
      enabled: True
      retain:
        days: 1
        mode: motion
      events:
        retain:
          default: 1
          mode: active_objects

The question is that in "Storage tab" I still read:

_Recordings & Snapshots =27.90 GB_

 What are those GB then?

**Comment by NickM-27** [comment]
The storage page shows what storage is used on those drives, not the storage that frigate itself is using.

**Reply by jokerigno** [reply]
so 27.90 is the whole space used by the entire VM (Home Assistant OS + Addons + Media?)

**Reply by NickM-27** [reply]
All space used on that drive.
